src/mongoose_listener.erl

```diff
@@ -61,7 +61,9 @@ stop_listener(Opts) ->
 %% Each listener module could be started more than once on different ports.
 -spec instrumentation([options()]) -> [mongoose_instrument:spec()].
 instrumentation(Listeners) ->
-    lists:usort([Spec || Listener <- Listeners, Spec <- listener_instrumentation(Listener)]).
+    %% c2s instrumentation is shared between Bosh, Websockets and TCP listeners
+    lists:usort([Spec || Listener <- Listeners, Spec <- listener_instrumentation(Listener)])
+    ++ mongoose_c2s:instrumentation().
```

Comment:
Just a note: we could alternatively call this in the BOSH, Websockets and C2S listeners? But it would be more code, and we usually want some kind of c2s functionality anyway, so I don't think it's needed.

Reply:
could be for sure from there.
I just was thinking, that we would not have many cases when none of them are configured, so could be just a common call anyway.

Could change and this:

```erlang
    instrument_helper:declared_events(mod_bosh, [])
    ++ instrument_helper:declared_events(mongoose_c2s, [global])
    ++ instrument_helper:declared_events(mongoose_c2s). %% For host_type()
 ```
 
 would become:
 
 ```erlang
     filter_events_by_host_type_or_global(host_type(), instrument_helper:declared_events(mod_bosh, [])).
```

Reply:
I think it's ok for now as it is.